recipes/pysparse/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pysparse" %}
+{% set version = "1.3_dev" %}
+{% set md5 = "3071ea517b5462888ceba71720f6f4a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # PyPI doesn't build
+  git_url: git://git.code.sf.net/p/pysparse/git
```

Comment:
Please, change that to use tarballs instead of repos

Please see:
https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Reply:
[The PySparse tarball is broken](https://sourceforge.net/p/pysparse/bugs/14/)